components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/model/recovery/v2/ResendRequest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2;
+
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.Property;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Request to resend the recovery code.
+ **/
+public class ResendRequest {
+
+    private String resendCode;
+    private List<Property> properties = null;
+
+    /**
+     * Resend code for the user.
+     **/
+    public ResendRequest resendCode(String resendCode) {
+
+        this.resendCode = resendCode;
+        return this;
+    }
+
+    /**
+     * Get resendCode
+     *
+     * @return resendCode
+     **/
+    @JsonProperty("resendCode")
+    public String getResendCode() {
+
+        return resendCode;
+    }
+
+    /**
+     * Set resendCode
+     *
+     * @param resendCode
+     **/
+    public void setResendCode(String resendCode) {
+
+        this.resendCode = resendCode;
+    }
+
+    /**
+     * (OPTIONAL) Additional META properties.
+     **/
+    public ResendRequest properties(List<Property> properties) {
+
+        this.properties = properties;
+        return this;
+    }
+
+    /**
+     * Get meta properties
+     *
+     * @return List<Property>
+     **/
+    @JsonProperty("properties")
+    public List<Property> getProperties() {
+
+        return properties;
+    }
+
+    /**
```

Comment:
Since we are adding comments to setter and getter in this class, shall we add comments to other model classes as well

Reply:
Removed the getter and setter comments from these files as well since it was not followed in other classes.

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/2c29350aad74d55ec4a6243007ac968e35cc02f1